### ISSUE TITLE
VM_default_mappings doesn't work, and Support remapping move key

### ISSUE THREAD

**Issue by Jaehaks** [mg979/vim-visual-multi#264]
Thank you for your efforts. 

My environment is
- Windows 10
- Nvim v0.9.4

1. First question) I saw about default mappings like this

>Default mappings are `permanent`, that is, always available, and applied when
Vim starts. Buffer mappings instead are applied per-buffer, when VM is started.
Permanent mappings, except <C-n>, can be disabled with:
`let g:VM_default_mappings = 0`

So I set this variable but the default mappings are alive when I enter VM mode 
![image](https://github.com/mg979/vim-visual-multi/assets/26200835/c8dce11f-823b-441f-bbdc-c2bb6293016e)
![image](https://github.com/mg979/vim-visual-multi/assets/26200835/cce5f12d-d8eb-4395-800f-32262969d853)

2. Second Question) Can I use j and k to move cursor instead of <Up> and <Down> (and <Left> , <Right>)? 
I cannot find the method about moving cursor. 
I think It can be convenient to move cursor like normal buffer and select with 'Add-Cursor-At-Pos'

**Comment by VICTORVICKIE** [comment]
setting default_mapping to zero and other config stuffs using vim.g.VM_* stuffs should rather be done on `init`, not on `config` for this plugin.

**Comment by Jaehaks** [comment]
@VICTORVICKIE 
Yes, I changed the config locaiton. it works except of VM-Find-Subword-Under as docs said. Thanks

**Comment by VICTORVICKIE** [comment]
you can take a look at my config, i have both finds in <C-d>
https://github.com/VICTORVICKIE/dotfiles/blob/main/nvim/lua/plugins/multi-cursor.lua

**Comment by pookdeveloper** [comment]
@VICTORVICKIE  please add to the doc, I have gone crazy to make it work haha

setting default_mapping to zero and other config stuffs using vim.g.VM_* stuffs should rather be done on init, not on config for this plugin.
